lms/static/sass/course/base/_base.scss

```diff
@@ -43,13 +43,13 @@ body {
   //background-image: $course-bg-image;
 }
 
-body, h1, h2, h3, h4, h5, h6, p, p a:link, p a:visited, a, label {
+body, h1, h2, h3, h4, h5, h6, p, label {
   @include text-align(left);
   font-family: $sans-serif;
 }
 
 table {
-  table-layout: fixed;
+  table-layout: auto;
```

Comment:
Instead of editing this rule, can you add a newer more specific rule below? Perhaps like this: 

```
// reset table layout to default inside xblocks
.xblock table {
    table-layout: auto;
}
```